meteor-app/imports/ui/assessment/assessment-add.js

```diff
@@ -18,7 +18,9 @@ const mapSchemaToState = schema => (
   schema
     .reduce((state, step) => {
       Object.keys(step.schema.properties)
-        .forEach(prop => state[prop] = undefined)
+        .forEach(prop => {
+          prop === 'services' || prop === 'parts' ? state[prop] = [] : state[prop] = undefined
```

Comment:
I'm sorry, I don't understand what this code is doing, and what you are trying to achieve.
You are mixing up logical expressions and ternary operators, and also doing assignments as well.
I think you need to simplify this code. Try and keep it to one idea per line. 
`undefined` is a strange thing, and shouldn't be used in assignments. Better to use either
`delete state[prop]` which will make it undefined, or `state[prop] = null` which will make it defined but set to null.